src/renderer/account/components/SendPanel/SendPanel.js

```diff
@@ -134,6 +151,10 @@ export default class AccountTxPanel extends React.Component {
     this.props.setReceiver(event.target.value);
   };
 
+  getAssets = (balances) => {
```

Comment:
Since this argument is always `this.props.balances`, I would suggest not passing an arg, and just accessing via `this.props` directly from this function.  i.e.:

```js
getAssets = () => {
  return mapValues(keyBy(values(this.props.balances), 'scriptHash'), 'symbol');
}
```